Classes/Service/IconProviderService.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types = 1);
+
+/*
+ * This file is part of the package bk2k/bootstrap-package.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace BK2K\BootstrapPackage\Service;
+
+use BK2K\BootstrapPackage\Events\ModifyIconProvidersEvent;
+use BK2K\BootstrapPackage\Icons\GlyphiconsProvider;
+use BK2K\BootstrapPackage\Icons\IconProviderInterface;
+use BK2K\BootstrapPackage\Icons\IoniconsProvider;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+final class IconProviderService
+{
+    private array $iconProvidersToAdd = [
```

Comment:
we should tag the providers as IconProvider instead of hardcoding them here - as initially discussed